transitions/registration.js

```diff
@@ -212,12 +212,20 @@ module.exports = {
         });
     },
     triggers: {
-        add_patient_id: function(options, cb) {
+        add_patient: function(options, cb) {
             // if we already have a patient id then return
             if (options.doc.patient_id) {
-                return;
+                return cb();
             }
-            module.exports.setId(options, cb);
+
+            async.series([
+                _.partial(module.exports.setId, options),
+                _.partial(module.exports.addPatient, options)
+            ], cb);
+        },
+        add_patient_id: function(options, cb) {
+            // Deprecated name for add_patient
+            module.exports.triggers.add_patient(options, cb);
```

Comment:
Maybe add `console.warn` here. It's not easily noticeable but we should at least try and notify people about deprecated triggers.